ioctl/newcmd/alias/aliasimport.go

```diff
@@ -46,77 +49,67 @@ type importMessage struct {
 	Unimported     []alias `json:"unimported"`
 }
 
-// NewAliasImportCmd represents the alias import command
-func NewAliasImportCmd(c ioctl.Client) *cobra.Command {
+// NewAliasImport represents the alias import command
+func NewAliasImport(c ioctl.Client) *cobra.Command {
+	var (
+		format      string
+		forceImport bool
+	)
+
 	use, _ := c.SelectTranslation(_importUses)
 	short, _ := c.SelectTranslation(_importShorts)
 	flagImportFormatUsage, _ := c.SelectTranslation(_flagImportFormatUsages)
 	flagForceImportUsage, _ := c.SelectTranslation(_flagForceImportUsages)
+	failToWriteToConfigFile, _ := c.SelectTranslation(_failToWriteToConfigFile)
 
 	ec := &cobra.Command{
 		Use:   use,
 		Short: short,
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			var err error
 			var importedAliases aliases
-			switch _format {
-			default:
-				return output.NewError(output.FlagError, fmt.Sprintf("invalid format flag%s", _format), nil)
+
+			switch format {
 			case "json":
 				if err := json.Unmarshal([]byte(args[0]), &importedAliases); err != nil {
-					return output.NewError(output.SerializationError, "failed to unmarshal imported aliases", err)
+					return errors.Wrap(err, "failed to unmarshal imported aliases")
 				}
 			case "yaml":
 				if err := yaml.Unmarshal([]byte(args[0]), &importedAliases); err != nil {
-					return output.NewError(output.SerializationError, "failed to unmarshal imported aliases", err)
+					return errors.Wrap(err, "failed to unmarshal imported aliases")
 				}
+			default:
+				return errors.New(fmt.Sprintf("invalid flag%s", format))
 			}
-			aliases := c.AliasMap()
+
 			message := importMessage{TotalNumber: len(importedAliases.Aliases), ImportedNumber: 0}
 			for _, importedAlias := range importedAliases.Aliases {
-				if !_forceImport && config.ReadConfig.Aliases[importedAlias.Name] != "" {
+				if !forceImport && c.Config().Aliases[importedAlias.Name] != "" {
 					message.Unimported = append(message.Unimported, importedAlias)
 					continue
 				}
-				for aliases[importedAlias.Address] != "" {
-					delete(config.ReadConfig.Aliases, aliases[importedAlias.Address])
-					aliases = c.AliasMap()
-				}
-				config.ReadConfig.Aliases[importedAlias.Name] = importedAlias.Address
+				c.SetAlias(args[0], importedAlias.Address)
```

Comment:
could `SetAliasAndSave` be used here?

Reply:
So we didn't use `SetAliasAndSave` here, right?